lib/components/link.vue

```diff
@@ -16,7 +16,7 @@
     export default {
         computed: {
             componentType() {
-                return (this.$router && this.to) ? 'router-link' : 'a';
+                return (this.$router && this.to && !this.disabled) ? 'router-link' : 'a';
```

Comment:
Possible,

Although the Bootstrap dropdown styling is also applied (making it look like menu items as well)

And I suppose the user could `v-if` the dropdown item to take it away if they are on the current route or want the item disabled. instead.  If the4 user uses `v-show="false"` I might have to alter the CSS selector for the keyboard navigation to include `[style*="display:none"]):not([style*="display: none"]` (or I could just document the ways to change the selector for these special cases).

I could remove this line for now, and just deal with the hiding of the menu.